src/main/kotlin/no/nav/tilleggsstonader/sak/ekstern/journalføring/HåndterSøknadService.kt

```diff
@@ -90,6 +92,14 @@ class HåndterSøknadService(
     }
 
     private fun finnStønadstypeForDagligReise(journalpost: Journalpost): Stønadstype {
+        if (!journalpost.harStrukturertSøknad()) {
+            return if (journalpost.tema == Tema.TSO.name) {
+                Stønadstype.DAGLIG_REISE_TSO
+            } else {
+                Stønadstype.DAGLIG_REISE_TSR
+            }
```

Comment:
Burde det være spesifikk sjekk og heller kaste feil om det gjelder noe annet enn TSO og TSR. Vi lytter vell ikke på det så den bør ikke treffe alt for ofte

Reply:
Det kommer aldri til å inntreffe siden vi filtrerer vekk andre tema ved lytting på topic 🤷